src/gossip/pull_request.zig

```diff
@@ -161,39 +167,38 @@ pub const CrdsFilterSet = struct {
             indexs.appendAssumeCapacity(i);
         }
 
-        const output_size = @min(set_size, max_size);
+        const n_filters = @min(set_size, max_size);
         const can_consume_all = max_size >= set_size;
 
         if (!can_consume_all) {
-
             // shuffle the indexs
-            var rng = std.rand.DefaultPrng.init(crds.getWallclockMs());
-            shuffleFirstN(rng.random(), usize, indexs.items, output_size);
+            var rng = std.rand.DefaultPrng.init(getWallclockMs());
+            shuffleFirstN(rng.random(), usize, indexs.items, n_filters);
 
             // release others
-            for (output_size..set_size) |i| {
+            for (n_filters..set_size) |i| {
                 const index = indexs.items[i];
                 self.filters.items[index].deinit();
             }
         }
 
-        var output = try ArrayList(CrdsFilter).initCapacity(alloc, output_size);
-        for (0..output_size) |i| {
+        var filters = try ArrayList(GossipFilter).initCapacity(alloc, n_filters);
+        for (0..n_filters) |i| {
             const index = indexs.items[i];
 
-            var output_item = CrdsFilter{
+            var filter = GossipFilter{
                 .filter = self.filters.items[index], // take ownership of filter
-                .mask = CrdsFilter.computeMask(index, self.mask_bits),
+                .mask = GossipFilter.computeMask(index, self.mask_bits),
                 .mask_bits = self.mask_bits,
             };
-            output.appendAssumeCapacity(output_item);
+            filters.appendAssumeCapacity(filter);
         }
 
-        return output;
+        return filters;
     }
 };
 
-pub const CrdsFilter = struct {
+pub const GossipFilter = struct {
```

Comment:
This is an improvement but it still feels a bit vague. What about PullFilter or GossipPullFilter?

Reply:
agreed (7590886)